docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -330,6 +327,45 @@ On touch devices, TestCafe emulates touch events instead of mouse events.
 
 Mouse event | Touch event
 ----------- | -------------
-`mousemove` (when hovering or dragging) | `touchmove` (dragging only)
+`mousemove` (when hovering or dragging) | `touchmove` (when dragging)
 `mousedown` | `touchstart`
 `mouseup`   | `touchend`
+
+## Interaction Requirements
+
+TestCafe actions can interact with elements if they satisfy the following conditions:
+
+* an element is within the page's [`body`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body) or in an [`<iframe>`](#work-with-iframes). The element can be invisible for the user. If the element is off-screen, TestCafe attempts to reach it with a scroll.
```

Comment:
Can we say the following?

`* an element is within the page's [`body`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body) or in an [`<iframe>`](#work-with-iframes). The element can be invisible to the user. If the element is off-screen, TestCafe uses a scroll action to attempt to locate it.`

Reply:
Not quite, because this implies that TestCafe has a scroll action, which isn't the case. Full list of actions available in TestCafe is listed [here](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/interact-with-the-page.html)

Does this work:
> an element is within the page's [body](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body) or in an [<iframe>](#work-with-iframes). The element can be invisible to the user. If the element is off-screen, TestCafe attempts to locate it with a scroll.

Reply:
It works, thanks. Please apply your suggestion.

Reply:
done